### ISSUE TITLE
[BUG] Stop russia! Stand with Ukraine!🇺🇦

### ISSUE THREAD

**Issue by stop-war-in-ukraine** [npm/npm-expansions#3747]
Block NPM for russians!

https://github.com/stop-war-in-ukraine/stop-russia-it

[pnpm 🇺🇦](https://pnpm.io/) - https://twitter.com/pnpmjs/status/1498306992577957890

Help!Help!Help! Stop war! Russi
<img width="438" alt="Screenshot 2022-03-01 at 22 40 40" src="https://user-images.githubusercontent.com/100472869/156246082-3efa4311-67af-4e80-b6d1-fd142b9f8cf6.png">
<img width="461" alt="Screenshot 2022-03-01 at 22 41 13" src="https://user-images.githubusercontent.com/100472869/156246091-e1d6d0ea-95d7-4e86-8294-1a864fe6cc59.png">
a kills civilians!!!!

**Comment by hackermondev** [comment]
> As an individual, I deeply sympathize with what the Ukrainian people are going through and wish nothing but peace and safety for Ukraine. I have personally expressed support on Twitter, and many Vue team / community members have done the same. However, I do not want Vue, an OSS project, to become a vehicle for political messages. We will do what we can as individuals, but not under the name of the project.

From https://github.com/vuejs/vue/pull/12483 probably same thing for npm

**Comment by a-a-GiTHuB-a-a** [comment]
While it's great that you support Ukraine, _I do not agree with adding political messages to this kind of project_.

**Comment by hedgerow512** [comment]
I dunno... While what's happening in Ukraine is sad, it's not the Russian citizens who are at fault. As a matter of fact, much of the Russian population disagrees what their government and military are doing.

**Comment by striped** [comment]
> I dunno... While what's happening in Ukraine is sad, it's not the Russian citizens who are at fault. As a matter of fact, much of the Russian population disagrees what their government and military are doing.

nonsense. 8 from 9 russians supports what happen.

**Comment by a-a-GiTHuB-a-a** [comment]
Because of he-who-must-not-be-named's STUPID PROPAGANDA. If they knew what was really happening, that would drop significantly.

**Comment by striped** [comment]
> Because of he-who-must-not-be-named's STUPID PROPAGANDA. If they knew what was really happening, that would drop significantly.

what it changes?

**Comment by hedgerow512** [comment]
> > I dunno... While what's happening in Ukraine is sad, it's not the Russian citizens who are at fault. As a matter of fact, much of the Russian population disagrees what their government and military are doing.
> 
> nonsense. 8 from 9 russians supports what happen.

Could you share that statistic with me please?

**Comment by striped** [comment]
> > > I dunno... While what's happening in Ukraine is sad, it's not the Russian citizens who are at fault. As a matter of fact, much of the Russian population disagrees what their government and military are doing.
> > 
> > 
> > nonsense. 8 from 9 russians supports what happen.
> 
> Could you share that statistic with me please?

Google stop working?
https://en.wikipedia.org/wiki/Public_image_of_Vladimir_Putin

**Comment by thatguywithdrip** [comment]
> 



> > > > I dunno... While what's happening in Ukraine is sad, it's not the Russian citizens who are at fault. As a matter of fact, much of the Russian population disagrees what their government and military are doing.
> > > 
> > > 
> > > nonsense. 8 from 9 russians supports what happen.
> > 
> > 
> > Could you share that statistic with me please?
> 
> Google stop working? https://en.wikipedia.org/wiki/Public_image_of_Vladimir_Putin

jumped to 69% who had a positive view of Putin in January 2022, and 71% who approved of the Russian president in February 2022 
(before the [2022 Russian invasion of Ukraine]

thats the only info about that that is there , note the „before russian invasion of ukraine” and remember they are brainwashed by russian propaganda. Locking npm to russian citizens will only harm the people who dont support the war or dont know what really is happening .

**Comment by striped** [comment]
War started 2014, not in 2022. If there are somebody who still doesn’t know what their country doing, they would never know.
For me personally, there is no people left. Human like looking beings who better keep on distance.

**Comment by a-a-GiTHuB-a-a** [comment]
Again, punishing those who don't know the whole story doesn't help.

**Comment by striped** [comment]
Depends. Punishing criminal doesn’t help victim. Punishing help the criminal realise responsibility for his/her crime.

In Ukraine people die just coz neighbour decided they should not live. We can not help them, they are gone. But we can punish those whose who allow that to happen. F..n russian, who explicitly or implicitly help PooTin to become who he is. If he won’t be stopped, this massacre continue and nobody knows who will be next. Do you want that happen with your country?

**Comment by thatguywithdrip** [comment]
> Depends. Punishing criminal doesn’t help victim. Punishing help the criminal realise responsibility for his/her crime.
> 
> In Ukraine people die just coz neighbour decided they should not live. We can not help them, they are gone. But we can punish those whose who allow that to happen. F..n russian, who explicitly or implicitly help PooTin to become who he is. If he won’t be stopped, this massacre continue and nobody knows who will be next. Do you want that happen with your country?

Punishing someone that doesnt even know what he did is fucking stupid my guy. And what did they even do apart from electing putin in the first place where he seemed like a good candidate. Remember you dont know how is it, you dont believe that something happened until you actually see it or someone shows evidence of it. And if you were in their shoes, what would you do? You would do nothing beacuse you would be scared. And then get people on social media call you not human

**Comment by striped** [comment]
They knew what they did. They fed the monster. 8 years is a plenty of time to see things around, isn’t it? You say, they were scared… They scared something they didn’t know?  And continue as nothing really happened. You managed stretch 69% just to 0. :) But it was not 69% who support Crimea annexation, it was much more. On my personal experience it is even worse: from all my russian “friends” only 1 was sorry. Just scared, yeah? And scarring shit out they voting for monster again and again, admiring him for terrorising neighbour nations: Georgia, Moldova, Lithuania… He made them greater. He brought them Crimea. Threaten Europe. Got them up from knees. Don’t tell me about propaganda, I knew how it works, I was born in USSR: they wish to be brainwashed. Everyone now blaming only putin, but forgot, he is their common creature. 
“Each nation deserves their ruler.”

**Comment by a-a-GiTHuB-a-a** [comment]
PUTIN HAS LITERALLY PUT LEGISLATION THAT ALLOWS HIM TO KEEP POWER UNTIL TWENTY THIRTY-SOMETHING. So it's _their fault_, huh?

**Comment by thatguywithdrip** [comment]
ok im done with this guy, someone please tell him that he is wrong

**Comment by striped** [comment]
thatguywithdrip, go f..k yourself

**Comment by hedgerow512** [comment]
Russian people are still people. They're humans just like Jews, Italians, or anyone else. That's what I was trying to say from my first comment. It quite bothers me when they all become scapegoated and dehumanized after their government does something drastic like invade a neighboring country. It's not like the Russian population has any control of what Putin does. That's why I don't think the Russian citizens should be punished, especially since Putin is showing no signs of stopping.

**Comment by striped** [comment]
Am not biologist, never try to discuss terminology. Also, you can’t dehumanise anyone in a chat… but we all may judge how other behave and make conclusions. It is not government kill civilians in occupied Ukrainian towns. It is not PooTin shell cities and villages. It is not them who came on neighbour territory terrifying all around. Right? And it is not them supported all evil that now happening. That is f..g russian do, usual people like you and me, roughly 7 if you met 8.
Do you like to be like them?

**Comment by hedgerow512** [comment]
Who are you trying to convince? Yourself or me?
Frankly, I can't take you seriously right now since it seems like you disregard all what anyone says of the matter and smother them in your bias instead. That's not going to convince anyone of anything, and it will certainly earn you a bad reputation.

**Comment by a-a-GiTHuB-a-a** [comment]
The grammar is not great either. The Russian soldiers may be doing wrong things, but _they're not the whole population and your "PooTin" is the one ordering them._

**Comment by striped** [comment]
Tell me about grammar. :)
You got lost? We spoke about russian, who support war in Ukraine and deserve isolation. I gave you facts, and you decided convince me that it is not true.

I don’t need convince you. Why? Who are you to speak about my reputation?

(These questions rhetorical, I don’t need your opinion.)

**Comment by hedgerow512** [comment]
You didn't give any facts. You just ranted the entire time. I gave reasons which were ignored or dismissed by you. That doesn't make what you say true.
It was your decision to comment on what I said in the first place, which I assumed would have become a productive conversation.

**Comment by striped** [comment]
Are you f..n russian? Usually that’s how they make conversation: just denied what doesn’t fit their baseline.

I gave you link on wiki, which has plenty of references. You just stated that people doesn’t responsible for authority they elect.
Which is childish nonsense. :)

Anyway, this conversation will be for people who can read, not for you.

**Comment by hedgerow512** [comment]
Lol you just committed a strawman, an ad hominem, and a false dilemma in one comment. Those are logical fallacies, in case you're not familiar.
Not to mention, the first line is quite a hasty generalization.

I never said the people are not responsible for the authority they elect. That's your strawman fallacy. I said the Russian people disagree with what their government is doing.
Your ad hominem is saying that this conversation is for people who can read. Obviously, I can read if I can respond.
Your false dilemma is assuming I'm Russian based off my perceived conversation style. It's not black and white like that.

I'm not Russian, dude. And if I was, you shouldn't believe that it would grant you a victory. I don't have to be Russian to have this mindset. I do, however, assume I'm well researched.

Therefore, argue the points properly!

Anyways, I still failed to find any information that 8 out of 9 Russians support Putin from the source you gave me. Even performing a Ctrl+F and searching, as well as further googling it, I couldn't find anything about it. The burden of proof is still on you.

Oh, and by the way... If I can make a personal jab, I think it's really funny that you say I can't read when clearly you're struggling with grammar. xD Sorry

**Comment by striped** [comment]
That is quote from wiki, I refer:
“After [EU and U.S. sanctions against Russian officials](https://en.m.wikipedia.org/wiki/List_of_individuals_sanctioned_during_the_2014_pro-Russian_unrest_in_Ukraine) as a result of the 2014 [Russian invasion of Ukraine](https://en.m.wikipedia.org/wiki/Russo-Ukrainian_War), Putin's approval rating reached 87 percent, according to a [Levada Center](https://en.m.wikipedia.org/wiki/Levada_Center) survey published on 6 August 2014.[[11]](https://en.m.wikipedia.org/wiki/Public_image_of_Vladimir_Putin#cite_note-11)”

87% ~ 8/9 and just for you, translate it into words:  8 russian from 9 support war in Ukraine.

how do you manage to do programming?

**Comment by a-a-GiTHuB-a-a** [comment]
That was _in 2014_. That is outdated information. Also, I agree that your grammar is low-quality. Can you show me the wiki and the exact place where it is?
(and also proof that _you_ didn't edit that in?)

**Comment by a-a-GiTHuB-a-a** [comment]
And do they really know what's going on? Or are they forced to watch Putin's mindless brainwashing _blah_ all day?

**Comment by striped** [comment]
Go f..k yourself, imbecile. Outdated moron with low quality reasoning.

**Comment by a-a-GiTHuB-a-a** [comment]
Caan you provide the burden of proof? _Proof_ that the Russians still approve of PooTin's war? (And not just a biased 5-person sample.)

**Comment by striped** [comment]
Imbecile, how you can make 87% from 5 person sample?
Just do your f..n research!

actually you, I’ll ignore you. You really boring idiot.

**Comment by a-a-GiTHuB-a-a** [comment]
As if I didn't know that. I meant that you might get 100% from a small sample and be like "OmG eVeRyOnE lOvEs HiM iM sO rIgHt"

**Comment by thatguywithdrip** [comment]
> As if I didn't know that. I meant that you might get 100% from a small sample and be like "OmG eVeRyOnE lOvEs HiM iM sO rIgHt"

ok m8 i think you got him, he stopped responding. GG

**Comment by a-a-GiTHuB-a-a** [comment]
No, I think they're just stubborn and dubbed us "not worth my valuable time."

**Comment by shockz09** [comment]
I think we should not add political messages to this kind of project.
Also to the people saying this will help, I don't get how will Ukraine People be helped by doing this?
Also its not the fault of Russian Citizens.

**Comment by Sly-Little-Fox** [comment]
Blocking NPM for Russia is pointless. A lot of Russian citizens support Ukraine. By blocking everything for Russian citizens, you won't help Ukraine. Regular citizens DIDN'T want that war and didn't vote for it.
P.S. Why in the world it's labeled as [BUG]?

**Comment by XFaonAE** [comment]
Wow, the stupidity of OP is quite clear, so clear that my hairline curled back and my jaw is dragging on the floor.
This as we call it could be a form of racism, for some reason, racism is looked down upon, *hmm i wonder why so*.

Turn ur screen off, and look at your self, think about the mmmmmistake you just made.
Poor Russias are constantly stareotyped and feed with false news about the "special military operation" as putin likes to call it.

There are many Russian developers who deserve to be treated better, its not just Americans and pretty much eery single other nationality that writes code.

**Comment by a-a-GiTHuB-a-a** [comment]
One group member being an idiot does not mean the entire group is the same kind of idiot. So goes with America, so goes with Russia.

**Comment by XFaonAE** [comment]
> One group member being an idiot does not mean the entire group is the same kind of idiot. So goes with America, so goes with Russia.

absokute respect my man, its all putins issue, hes not well in the head

**Comment by XFaonAE** [comment]
Yk, many of the russian citizans who were able to receive accurate news, they them selfs hate putin. Putin is the one to blame, i mean he went as far as to steal money form his VERY OWN citizens by faking tax fraud, even tho they were completly innocent

**Comment by XFaonAE** [comment]
The only way for this to end is for Putin's assassination, but thats failed as many have tried. All we can do is pray.

**Comment by Khazbs** [comment]
Yes, we despise the current president of our country and we do want to stop the war. We protest and riot relentlessly, only to get tortured, go to prison, lose our schools, jobs and sometimes lives. We already receive enough punishment from our government just for being born in Russia. Banning civil technologies in our country doesn't help anyone.
